Dockerfile

```diff
@@ -125,15 +125,14 @@ WORKDIR /app
 RUN apt-get update \
   && apt-get -y --no-install-recommends install \
     ca-certificates \
-    build-essential=12.9 \
```

Comment:
Can you fix the big comment above? Possibly we can keep the bulk of the comment as I still think it's quite useful, but we can say we are not fixing the versions at present, due to issues with the debian package manage taking down outdated versions.